src/ui/components/pages/importKeystore/importKeystore.tsx

```diff
@@ -49,6 +49,9 @@ type ExchangeKeystoreAccount = {
   | {
       userType: 'wavesKeeper';
     }
+  | {
+      userType: 'keeperWallet';
```

Comment:
`ExchangeKeystoreAccount` defines the wave.exchange import file format, so I don't think this field will be renamed in the future.
So maybe it's not necessary?